apps/site/src/app/apply/sections/ApplyConfirmation/ConfirmationDetails.tsx

```diff
@@ -35,13 +35,13 @@ export default async function ConfirmationDetails({
 			<hr className="mt-5 w-full h-0.5 bg-[#432810]" />
 			<p className="text-display">
 				Interested in helping out instead? Consider applying to be a{" "}
-				<Link className="underline" href="/mentor">
+				<a className="underline" href="/mentor">
```

Comment:
Issue: Instead of using `a` tags, you can set the `prefetch` attribute to false. See the [`next/link`](https://nextjs.org/docs/app/api-reference/components/link) documentation for an example.

Reply:
I tried just disabling prefetch and that stopped the cors error from happening on the application preface. 

However, clicking the next link raised another cors error, but still allowed me to access the Google form.

Reply:
I see. We can stick to `a` tags then, especially since the form isn't a page on our site.